pyensembl/normalization.py

```diff
@@ -38,17 +38,13 @@ def normalize_chromosome(c):
     elif result == "":
         raise ValueError("Chromosome name cannot be empty")
 
-    # only strip off lowercase chr since some of the non-chromosomal
-    # contigs start with "CHR"
-    if result.startswith("chr"):
-        result = result[3:]
-
-    # just in case someone is being lazy, capitalize "M", "MT", X", "Y"
-    result = result.upper()
-
-    # standardize mitochondrial genome to be "MT"
-    if result == "M":
-        result = "MT"
+    if result.startswith("chr") and "_" not in result:
```

Comment:
thought for the future: normalize_chromosome could be aware of its Pyensembl release and normalize it to that set of contig names?

Reply:
I was thinking something similar! It's a little tricky since the normalization also gets applied to GTF file contigs which are kind of a lower level concept but I suspect we could figure out a sane rewiring.